fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1723,8 +1723,12 @@ showWhiteListStatement
 
 // ------------------------------------------- Export Statement --------------------------------------------------------
 
+asyncStatement
+    : SYNC | ASYNC
+    ;
+
 exportStatement
```

Comment:
It's better to add a session variable like `sync_mode`. Adding `SYNC|ASYNC` in the command suffix is strange.

Reply:
has changed to use `(WITH (SYNC | ASYNC) MODE)?` like [ANALYZE TABLE](https://docs.starrocks.io/zh-cn/latest/sql-reference/sql-statements/data-definition/ANALYZE%20TABLE#%E8%AF%AD%E6%B3%95) and [REFRESH MATERIALIZED VIEW](https://docs.starrocks.io/zh-cn/latest/sql-reference/sql-statements/data-manipulation/REFRESH%20MATERIALIZED%20VIEW#%E8%AF%AD%E6%B3%95)